src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PDB/CustomDebugInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.VisualStudio.SymReaderInterop
+{
+    internal struct CustomDebugInfo
```

Comment:
MethodDebugInfo.  I've renamed the exist MethodDebugInfo test helper to MethodDebugInfoBytes.